hummingbot/connector/exchange/kraken/kraken_in_fight_order.py

```diff
@@ -0,0 +1,131 @@
+import copy
+import math
+from decimal import Decimal
+from typing import Any, Dict, Optional, Tuple
+
+from hummingbot.core.data_type.common import OrderType, PositionAction, TradeType
+from hummingbot.core.data_type.in_flight_order import InFlightOrder, OrderState, TradeUpdate
+
+
+class KrakenInFlightOrder(InFlightOrder):
```

Comment:
why we need an specific implementation of InFlightOrder?

Reply:
Because in the kraken document, there is a “userref” attribute (equivalent to “client_order_id”), but it is of type int32 and is not unique.
This attribute was added to the older version of the connector, KrakenInFlightOrder, to replace much of the work of client_order_id.
Of course, just now, in the latest hummingbot, I discovered that we can use the "get_new_numeric_client_order_id" method
Of course, just now, in the latest hummingbot, I discovered that you can use the get_new_numeric_client_order_id method.
If necessary, I can modify the code to replace the userref attribute of KrakenInFlightOrder with ”get_new_numeric_client_order_id“ method

Reply:
![3571709707449_ pic](https://github.com/hummingbot/hummingbot/assets/29802081/fb1d8597-b8d8-422e-a146-620b02f90b80)